bin/gce-delete-instance

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
 
-uage="Usage: `basename $0` <instance_name> [-q | --quite]"
+delete-instance() {
+  uage="Usage: `basename $0` <instance_name> [-q | --quite]"
 
-instname=$1
-if [ -z "$instname" ]; then
+  instname=$1
+  if [ -z "$instname" ]; then
+    echo $usage
+    exit
+  fi
+
+  while [[ $# -gt 0 ]]
+  do
+  key="$1"
+
+  cmd="gcloud compute instances delete $instname"
+
+  case $key in
+    -q|--quite)
+    cmd="$cmd -q"
+    ;;
+    -h|--help)
+    help="1"
+    ;;
+    *)
+      # unknown option
+    ;;
+  esac
+  shift # past argument or value
+  done
+
+  if [ -n "$help" ]; then
+    echo $usage
+    exit
+  fi
+
+  $cmd
+  instdir=$SCGCE/var/run/$instname
+  if [ -d $instdir ]; then rm -rf $instdir; fi
+}
+
+usage="Usage: `basename $0` [instnaces, ex. \"instance1 instance2 ...\"] [-q | --quite]"
```

Comment:
Typo: `instnaces`.

It could be clearer to use `[instance1 [instance2 [...]]]`.

If you prefer to use the current form, `ex.` should be `e.g.`.